server/service/src/sync/program_schemas/hiv_care_encounter_ui_schema.json

```diff
@@ -513,8 +556,297 @@
         },
         {
           "type": "Control",
-          "label": "HCV",
-          "scope": "#/properties/tbHcv/properties/hcv"
+          "label": "TB-Status:",
+          "scope": "#/properties/tuberculosis/properties/tbStatus"
+        }
+      ]
+    },
+    {
+      "type": "Category",
+      "label": "Index Testing",
+      "options": { "icon": "index-testing" },
+      "elements": [
+        {
+          "type": "Control",
+          "label": "Index Testing Offered:",
+          "scope": "#/properties/indexTesting/properties/indexTestingOffered"
+        },
+        {
+          "type": "Control",
+          "label": "IPV Risk?",
+          "scope": "#/properties/indexTesting/properties/ipvRisk"
+        },
+        {
+          "type": "Control",
+          "label": "Index Testing Accepted?",
+          "scope": "#/properties/indexTesting/properties/indexTestingAccepted"
+        }
+      ]
+    },
+    {
+      "type": "Category",
+      "label": "HCV / CrAg",
+      "options": { "icon": "liver" },
+      "elements": [
+        {
+          "type": "Control",
+          "label": "HCV:",
+          "scope": "#/properties/hcvCrag/properties/hcv"
+        },
+        {
+          "type": "Control",
+          "label": "CrAg Result:",
+          "scope": "#/properties/hcvCrag/properties/crAgResult"
+        },
+        {
+          "type": "Control",
+          "scope": "#/properties/hcvCrag/properties/crAgTitre"
+        }
+      ]
+    },
+    {
+      "type": "Category",
+      "label": "General",
+      "options": { "icon": "general" },
+      "elements": [
+        {
+          "type": "Control",
+          "label": "Referred to:",
+          "scope": "#/properties/general/properties/referredTo"
+        },
+        {
+          "type": "Control",
+          "label": "Visit Type:",
+          "scope": "#/properties/general/properties/visitType"
+        },
+        {
+          "type": "Control",
+          "scope": "#/properties/general/properties/location"
+        }
+      ]
+    },
+    {
+      "type": "Category",
+      "label": "Risk Behaviour",
+      "options": { "icon": "document" },
+      "elements": [
+        {
+          "type": "Control",
+          "label": "Previous HIV Test Result:",
+          "scope": "#/properties/riskBehaviour/properties/previousHIVTestResult",
```

Comment:
This needs to be removed...

Reply:
Opps thought I removed it sorry 🙏🏻  [ecdc3d6](https://github.com/openmsupply/open-msupply/pull/1221/commits/ecdc3d63d7a1d7dbd460b2ee800688a9c361c9db)